traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/BookingItemSortOrder.java

```diff
@@ -0,0 +1,41 @@
+package com.guestlogix.travelercorekit.models;
+
+import androidx.annotation.NonNull;
+
+public enum BookingItemSortOrder {
+    RECOMMENDED("", "Recommended"),
+    PRICE_LOW_TO_HIGH("Asc", "Low to High"),
+    PRICE_HIGH_TO_LOW("Desc", "High to Low");
+
+    private final String sortValue;
+    private final String sortTitle;
+
+    BookingItemSortOrder(String sortValue, String sortTitle) {
+        this.sortValue = sortValue;
+        this.sortTitle = sortTitle;
+    }
+
+    public static BookingItemSortOrder fromSortText(String value) throws IllegalArgumentException {
+        switch (value) {
+            case "Recommended":
```

Comment:
Don't actually need "Recommended" as long as you're preselecting the default sort option (i.e. Price: Low to high)

Reply:
i think i need a none atleast. because what if user didnt want to sort by title ?

Reply:
Ensure at least one sorting option must be selected.  Should not be possible to not have some type of sorting

Reply:
If you want to model a "none" as a catch all just in case, then do it, but it probably makes more sense to force at least one of the valid options to be set

Reply:
oh i just saw yuur commetns already deleted mine because i fixed that. we already using null checking for default sort. if its null we dont send it so i can even use null as default though i wont but client programer might and its not a problem now